ui/app/pages/asset/asset.scss

```diff
@@ -0,0 +1,50 @@
+.asset {
+  &__container {
+    overflow-y: auto;
+    background-color: white;
+
+    &::-webkit-scrollbar {
```

Comment:
We should probably add `scrollbar-width: none;` here as well? Currently the page has a scrollbar in Firefox.

Reply:
Huh, that rule shouldn't be there at all. There shouldn't be an scrollbar here because the whole page scrolls instead. I'll remove this.

Reply:
This has been addressed in d07d54b3516977b53a9dab949d0dbf6fcbb9820b

Reply:
Ah, so we want a scrollbar on the whole page?

Reply:
This was Firefox before your most recent commit:
![image](https://user-images.githubusercontent.com/25517051/83430236-249a7b00-a3ea-11ea-9809-fbc508e7fd51.png)

Reply:
I believe this is a pre-existing discrepancy between Chrome and Firefox. That scrollbar is not there due to the `asset__container` element as far as I can tell.

Reply:
I was able to reproduce this same scrollbar on the home page as well by adding many assets.

I don't entirely understand why the scrollbar is hidden in Chrome, but not Firefox 🤔 There is `webkit-scrollbar` rule on `main-container` to disable the scrollbar on Chrome, but adding `scrollbar-width: none` to the same class seemed to have no effect. I had to add `scrollbar-width: none` to `.app` to disable it for some reason.